sympy/stats/crv.py

```diff
@@ -41,6 +49,27 @@ class SingleContinuousDomain(ContinuousDomain, SingleDomain):
     Represented using a single symbol and interval.
     """
     def compute_expectation(self, expr, variables=None, **kwargs):
+        """
+        Computes the expectation in Given Domain
+
+        Parameters
+        ==========
+
+        self,
```

Comment:
`self` need not be documented.

And other should follow the format, for example
https://numpydoc.readthedocs.io/en/latest/example.html#example

```
    Parameters
    ----------
    var1 : array_like
        Array_like means all those objects -- lists, nested lists, etc. --
        that can be converted to an array.  We can also refer to
        variables like `var1`.
    var2 : int
        The type above can either refer to an actual Python type
        (e.g. ``int``), or describe the type of the variable in more
        detail, e.g. ``(N,) ndarray`` or ``array_like``.
    long_var_name : {'hi', 'ho'}, optional
        Choices in brackets, default first when optional.
```
or everything will likely be rendered as bold characters.